app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -340,6 +345,13 @@ public void onBookFound(LibraryNetworkEntity.Book book) {
           zimManageActivity.runOnUiThread(() -> {
             Log.i("Scanner", "File Search: Found Book " + book.title);
             mFiles.add(book);
+            String lastOpenedBook = sharedpreferences.getString(LAST_BOOK_OPENED,"null");
+            for(LibraryNetworkEntity.Book mybook : mFiles){
```

Comment:
Don't use Hungarian notations.

Reply:
Which variable are you talking about? mFiles existed before this commit.

Reply:
`mFiles`, you can rename it to `files`.